example/core_tutorial/12_halo_exchange/CMakeLists.txt

```diff
@@ -12,5 +12,5 @@
   add_executable(HaloExchange halo_exchange_example.cpp)
   target_link_libraries(HaloExchange cabanacore)
   add_test(NAME Core_tutorial_12 COMMAND ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG}
-    ${MPIEXEC_MAX_NUMPROCS} ${MPIEXEC_PREFLAGS} HaloExchange ${MPIEXEC_POSTFLAGS})
+          ${MPIEXEC_MAX_NUMPROCS} ${MPIEXEC_PREFLAGS} $<TARGET_FILE:HaloExchange> ${MPIEXEC_POSTFLAGS})
```

Comment:
```suggestion
    ${MPIEXEC_MAX_NUMPROCS} ${MPIEXEC_PREFLAGS} $<TARGET_FILE:HaloExchange> ${MPIEXEC_POSTFLAGS})
```

Reply:
Waiting to see if Christoph request more changes before I apply that suggestion to avoid re-triggering the CI more than necessary.